### PR TITLE
Remove marketing blocks from PRODUCT mode home page

### PR DESCRIPTION
**Pull request by fiskus**
Screenshot from 2022-04-05 12-21-25

### CODE REVIEW

File: catalog/app/containers/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -389,13 +389,13 @@ function useLinks(): LinkDescriptor[] {
       label: 'URI',
       path: paths.uriResolver,
     },
-    { href: URLS.docs, label: 'Docs' },
+    cfg.mode !== 'PRODUCT' && { href: URLS.docs, label: 'Docs' },
```
Comment (nl0) [inline catalog/app/containers/NavBar/NavBar.tsx:392]: i think we should keep the documentation link

File: catalog/app/website/pages/Landing/Landing.js [hunk 2]
```diff
@@ -24,23 +24,27 @@ import UseQuilt from './UseQuilt'
...
-      {cfg.mode !== 'LOCAL' && <Platform />}
-      {cfg.mode !== 'LOCAL' && <Highlights />}
+      {cfg.mode === 'LOCAL' && <LocalMode />}
+      {showMarketingBlocks && <Showcase />}
+      {showMarketingBlocks && <UseQuilt />}
+      {showMarketingBlocks && <Logos />}
+      {showMarketingBlocks && <CaseStudies />}
+      {showMarketingBlocks && <Testimonials />}
+      {showMarketingBlocks && <Platform />}
+      {showMarketingBlocks && <Highlights />}
```
Comment (nl0) [inline catalog/app/website/pages/Landing/Landing.js:45]: we could wrap these into a fragment and keep only one check before it, e.g.:
```
{showMarketingBlocks && (
  <>
    <Showcase />
    <UseQuilt />
    ...
  </>
)}
```
no big deal tho, up to you

**Review by nl0 — APPROVED** [review]

File: catalog/app/containers/NavBar/NavBar.tsx (hunk 1 above)
Comment (fiskus) [reply catalog/app/containers/NavBar/NavBar.tsx:392]: Aneesh asked to remove it. I thought we should keep it too.

But after removing all marketing blocks docs link in the footer become visible now, and probably redundant

Comment (fiskus) [reply catalog/app/containers/NavBar/NavBar.tsx:392]: Also, we have docs links in-place where something goes wrong and user should want to check the docs :)

**Pull request merged**